### DISCUSSION TITLE
download artisan 2.4.2

### DISCUSSION THREAD

**Discussion by mangga88d** [artisan-roaster-scope/artisan#1162] [Q&A]
do you have files for artisan 2.4.2 for windows. i am trying to get the apps for that. seems in the assets only available for the source code. 
any link would help.

thank  you

**Comment by MAKOMO** [comment]
Why not take the latest release which is v2.8.2?

**Reply by mangga88d** [reply]
i can not use because of the thermocouple readers. the drivers can work no problem with 2.4.2 version. otherwise when tried to switch ON on the artisan software, the thermocouple readers display keep restart itself.

**Reply by MAKOMO** [reply]
So you say your device works correctly with v2.4.2, but not with the current v2.8.2? Which device are you connecting?

**Reply by mangga88d** [reply]
i am connecting to thermocouple readers which is made locally. i bought it online. i asked the person for 2.4.2 but he does not have it no more. he asked me to use latest version, but latest version has this issue.

**Reply by MAKOMO** [reply]
If this reader is not officially supported I fear we cannot help you here. You need to ask that person to make his device compatible with the latest version. Artisan v2.4.2 is no longer available. Sorry.

**Comment by MAKOMO** [comment]
One Idea: We are currently preparing the next release of Artisan. In case your hardware is communicating via MODBUS the current Continuous Build might work with your hardware again as we made some improvements to the MODBUS layer. You can find the unsigned builds here:

https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous

Please report!

**Comment by mangga88d** [comment]
thank you for your help makomo, appreciate it, i will try next time, currently i just bought new hardware that works. may be next time will be able to use it again. 
thank you so much
